### DISCUSSION TITLE
Bravo Connection Not Available Error : HTTP/500 domain user does not have permission for Discover-Methode

### DISCUSSION THREAD

**Discussion by jalili2006** [sql-bi/Bravo#792] [Q&A]
After installing Bravo and trying to attach it to Power BI Desktop, we encountered an issue: Bravo Power BI tool error 500 - the user does not have permission to access the Discover method. In the domain controller, what permissions does the user need to connect Bravo to Power BI, user already has domain-user group .

**Comment by jalili2006** [comment]
![Github_Error(2)](https://github.com/sql-bi/Bravo/assets/7502093/98107e56-4ea2-4b82-977f-13c0e9912110)

**Comment by jalili2006** [comment]
user already joined to Domain-User Group

**Comment by albertospelta** [comment]
@jalili2006 It looks like you are using an older version of the application (1.0.2). The issue you're reporting should have been fixed in one of the subsequent versions. Could you please try updating Bravo to the latest version [here](https://github.com/sql-bi/Bravo/releases/latest) and let us know if the problem persists?

**Reply by jalili2006** [reply]
Bravo Ver.1.0.8
PowerBI Desktop 2.1
Thanks for info , one possibility i did not check is Client use Power BI on Cloud and in Power BI Service Administrator - Workspace Permissions should be member of Contributor, I think this issue back to permission on user privilege in Web portal or Firewall services for outbound and inbound for Bravo.exe App

**Reply by albertospelta** [reply]
If the issue still persists after updating to version 1.0.8, it is likely due to the workspace permissions in Power BI Service.
